joomla/joomla-cms
# [5.1] Update deleted files list in script.php for 5.1.0-beta2

Pull Request for Issue # .

### Summary of Changes

This pull request (PR) updates the lists of files and folders to be deleted on update in file `administrator/components/com_admin/script.php` by the changes from PR #42954 in the 5.1-dev branch in preparation for the upcoming 5.1.0-beta2 release.

### Testing Instructions

Code review, or if you want to do a real test:

Update the latest 5.1 nightly build to the patched package created by drone for PR 42954 to get the actual result.

You can find that update package or custom update URL here: https://artifacts.joomla.org/drone/joomla/joomla-cms/5.1-dev/42954/downloads/74344/

Check if the files and folders added by this PR here are present.

Result: See actual result below.

Now update to the patched package created by drone for this PR here.

You can find that update package or custom update URL here: https://artifacts.joomla.org/drone/joomla/joomla-cms/5.1-dev/42955/downloads/74345/

Check if the files and folders added by this PR here are present.

Result: See expected result below.

### Actual result BEFORE applying this Pull Request

The files and folders are still present.

### Expected result AFTER applying this Pull Request

The files and folders have been deleted with the update.

### Link to documentations
Please select:
